src/resources/Sources/SourcesMappings/SourcesMappingsInterfaces.ts

```diff
@@ -0,0 +1,43 @@
+import {FieldTypes, MappingModelKind} from '../../Enums';
+
+export interface GetSourceMappingParams {
+    organizationId?: string;
+}
+
+export interface MappingsConfiguration {
+    common: {
+        rules: MappingRule[];
+    };
+    types: Array<{
+        rules: MappingRule;
+        type: string;
```

Comment:
There is supposed to be an `id: string` in there according to the swagger doc.

Reply:
This has been updated in the latest amend :+1: